src/index.html

```diff
@@ -1,13 +1,53 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img src="./images/imac.jpeg" alt="Computer" class="card__image">
+
+      <div class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+
+      <div class="card__code">Product code: 195434</div>
+
+      <div class="card__rating">
+        <div class="stars stars--4">
+          <span class="stars__star">
+          </span>
```

Comment:
```suggestion
          <span class="stars__star"></span>
```
not necessary to start closing tag on new line